net/mdns-repeater/src/opnsense/service/templates/OPNsense/MDNSRepeater/mdnsrepeater

```diff
@@ -1,6 +1,9 @@
 {% if helpers.exists('OPNsense.MDNSRepeater.enabled') and OPNsense.MDNSRepeater.enabled == '1' %}
 {% from 'OPNsense/Macros/interface.macro' import physical_interface %}
 mdns_repeater_enable="YES"
+{% if helpers.exists('OPNsense.MDNSRepeater.enablecarp') and OPNsense.MDNSRepeater.enablecarp == '1' %}
+required_files="/var/run/mdns-repeater/CARP_MASTER"
```

Comment:
sorry to be nitpicking. I think the directory trick was only done because the other plugin already used a directory for itself but maybe we can avoid this bit of complexity by doing the following?

```suggestion
required_files="/var/run/mdns-repeater.CARP_MASTER"
```